pkg/provisioner/provisioner.go

```diff
@@ -243,6 +235,12 @@ func getProvisioner(provider, region string) (automation.Automation, error) {
 			return nil, err
 		}
 		return cloudProvider, nil
+	case "aws":
+		cloudProvider, err := aws.NewAWS(region, os.Getenv("AWS_ACCESS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"), os.Getenv("AWS_SESSION_TOKEN"))
```

Comment:
@cpanato, can you add the credential part to the README.md?

Reply:
ahh of course, I've forgot the readme :( sorry will do that